src/libraries/ERC7739Utils.sol

```diff
@@ -27,30 +25,30 @@ library ERC7739Utils {
     /// @param contentsHash The hash of the contents, per EIP-712
     /// @param domainBytes The encoded domain bytes from EIP-5267
     /// @param appSeparator The app's domain separator
-    /// @param contentsName The top level type, per EIP-712
-    /// @param contentsType The full type string of the contents, per EIP-712
+    /// @param contentsDescr The full type string of the contents, per EIP-712
     function toNestedTypedDataSignHash(
         bytes32 contentsHash,
         bytes memory domainBytes,
         bytes32 appSeparator,
-        string memory contentsName,
-        string memory contentsType
+        string calldata contentsDescr
```

Comment:
Why do we change this?

Reply:
I'm actually not sure, I might have been getting stack too deep before, but now it works. I'll revert it back and it'll be cleaner!